dockertest/docker_daemon.py

```diff
@@ -153,14 +153,22 @@ def restart():
     return _systemd_action('restart')
 
 
+def systemd_show(prop):
+    """
+    Runs 'systemctl show --property=ARG' for given arg.
+    Returns the value as a string.
+    """
+    cmd_result = _systemd_action('show --property=%s' % prop)
+    stdout = cmd_result.stdout
+    if not stdout.startswith(prop + '='):
+        raise RuntimeError("Unexpected output from %s: expected %s=XXXX,"
+                           " got '%s'" % (cmd_result.command, prop, stdout))
+    return stdout[(len(prop) + 1):].strip()
+
+
```

Comment:
Naa, I do mean ``_systemd_action()`` (the private function).  If you mock that, then testing all the other functions which call it should be easy.  My thought was to have a test method run through a list of mappings for function-name, input-arguments, expected return, and expected wrapper-arguments (to the mocked ``_systemd_action()``.  Just a cheap idea though, do whatever is easier

> python testing always takes me a while; I'll have something ready tomorrow.

Mannn, don't you know it!  Testing is gotta be like 10-times harder than writing the subject-code itself, no doubt!  Take your time, there's no rush.  I don't care if unittest code is pretty, so long as it gets the job done. Fugly and duct-tape covered is perfectly acceptable 😀